examples/load_balancer/load_balancer.c

```diff
@@ -95,6 +95,13 @@ struct loadbalance {
 
         /* config file */
         char *cfg_filename;
+        
+        /* LB policy */
+        char *policy;
+
+        /* structures to store server weights */
+        int *weights;
+        int total_weight;
```

Comment:
your data structure here will allow us to represent weighted load balancers, but there could be other policies we could come up with in the future that might need totally different data. For now this is fine, but you should think about how you would make this data structure generic so it works for all LBs (but still allows customization by each policy in some way)